GooglePlayInstant/Editor/AssetBundleVerifierWindow.cs

```diff
@@ -86,47 +93,56 @@ private void GetAssetBundleInfoFromDownload()
                 // all objects that were loaded from this bundle.
                 bundle.Unload(true);
             }
-
-            // Turn request to null to be prepared for next call
-            _webRequest.Dispose();
         }
 
         private double ConvertBytesToMegabytes(ulong bytes)
         {
             return bytes / 1024f / 1024f;
         }
 
-        //TODO: fix malformed url behavior
         private void Update()
         {
+            if (_webRequest == null || !_webRequest.isDone)
+                return;
+            
             // Performs download operation only once when webrequest is completed.
-            if ((_webRequest != null) && (_webRequest.isDone))
-            {
-                GetAssetBundleInfoFromDownload();
-                Repaint();
-            }
+            GetAssetBundleInfoFromDownload();
+            Repaint();
+            
+            // Turn request to null to signal ready for next call
+            _webRequest.Dispose();
+            _webRequest = null;
         }
 
-        //TODO: add a loading state so client knows that some loading is going on behind the scenes.
         private void OnGUI()
         {
-            AddVerifyComponentInfo("AssetBundle Download Status:",
-                _assetBundleDownloadIsSuccessful ? "SUCCESS" : "FAILED");
+            if (_webRequest!= null && !_webRequest.isDone)
```

Comment:
Nit: run the Code -> Reformat Code to fix little issues like on this line

Reply:
My code reformatter doesn't catch these issues? I ran it a couple times before submitting? Anyways, not sure what nit or line you're specifically referring to since my reformatter isn't changing the code. 

I added brackets if that's what you meant.